### ISSUE TITLE
Object rest operator works incorrectly on assigment

### ISSUE THREAD

**Issue by KSXGitHub** [gkz/LiveScript#985]
### LiveScript Code

```livescript
const obj = {abc: 123, def: 456, ghi: 789}
const {abc, ...rest} = obj
console.log rest
```

### Expected Output

```javascript
{def: 456, ghi: 789}
```

### Received Output

```javascript
{}
```

**Comment by rhendric** [comment]
Duplicate of #941, should be fixed in #958—have you tried using master?

**Comment by KSXGitHub** [comment]
I think it's better to publish new version (v1.5.1) to npm every time a bug is fixed rather than just leave it at master branch.
